### DISCUSSION TITLE
Step by step installation for Surfshark

### DISCUSSION THREAD

**Discussion by Kn0xsec** [haugene/docker-transmission-openvpn#2679] [Q&A]
Hello, 

I just bought Surfshark VPN and wanted to utilize it with Transmission Docker. According to the description for this repo, Surfshark appears to be a supported VPN, however I can't find any step-by-step instructions on how to deploy the container with the Surfshark VPN config file. I'm new to this setup and don't want to mess it up, so please share any resources you have on how to install it appropriately.

**Comment by pkishino** [comment]
Please read the documentation linked in the read me :)

**Reply by pkishino** [reply]
Afaik there is no special config needed for that provider.. just follow default
